nautobot/dcim/migrations/0033_remove_region_and_site.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.17 on 2023-02-14 18:03
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("circuits", "0011_remove_site_foreign_key_from_circuit_termination_class"),
+        ("dcim", "0032_remove_site_foreign_key_from_dcim_models"),
+        ("extras", "0063_remove_site_and_region_attributes_from_config_context"),
+        ("ipam", "0018_remove_site_foreign_key_from_ipam_models"),
+        ("virtualization", "0016_remove_site_foreign_key_from_cluster_class"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="site",
+            name="region",
+        ),
+        migrations.RemoveField(
+            model_name="site",
+            name="status",
+        ),
+        migrations.RemoveField(
+            model_name="site",
+            name="tags",
+        ),
+        migrations.RemoveField(
+            model_name="site",
+            name="tenant",
+        ),
```

Comment:
Interesting that these fields need to be deleted individually before deleting the model as a whole. Wonder why that is.

Reply:
Maybe because they are ForeignKey fields and ManytoMany fields? Im not too sure.